sys/net/gnrc/sock/udp/gnrc_sock_udp.c

```diff
@@ -346,6 +346,14 @@ ssize_t sock_udp_sendv_aux(sock_udp_t *sock,
         src_port = sock->local.port;
         memcpy(&local, &sock->local, sizeof(local));
     }
+#if IS_USED(MODULE_SOCK_AUX_LOCAL)
+    /* user supplied local endpoint takes precedent */
+    if ((aux != NULL) && (aux->flags & SOCK_AUX_GET_LOCAL)) {
```

Comment:
Maybe `SOCK_AUX_SET_LOCAL` or `SOCK_AUX_USE_LOCAL` or `SOCK_AUX_HAVE_LOCAL` instead? It could have the same numerical value as `SOCK_AUX_GET_LOCAL`.

I can also imagine that someone wants to set from an unbound socket, let the network stack decide which IP address is the most suitable to send from, but than get the address actually use. For that, the name `SOCK_AUX_GET_LOCAL` would again make most sense. So IMO it is better to leave this name really for just retrieving the local address.